src/components/molecules/FooterSNSIcons/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { StyleSNSIconsWrapper } from "./styles"
+import SocialIcon from "../../atoms/SNSIcon"
+import translationsEn from "../../../../config/translations/en.json"
+
+const SNSList = translationsEn.social;
+
+const FooterSNSIcons: React.FC = () => {
+    return (
+        <StyleSNSIconsWrapper>
+            {SNSList.map(item => (
+                <SocialIcon serviceName={`${item.service.toLowerCase()}`} />
```

Comment:
re: q1 - I use toLowerCase so I can feed it to the switch statement in the individual icon component.  I don't think you can use CSS since it decides what icon to display based on a string, but is there a better way?